site/config-overrides.md

```diff
@@ -1,108 +1,108 @@
-# Configuration Overrides
+# Configuration overrides
 
-**PLEASE NOTE**:  This page is a work in progress. 
+**PLEASE NOTE**:  This page is a work in progress.
 
 ---
 # Overview
 
-Use configuration overrides to customize a WebLogic domain home configuration without modifying the domain's actual config.xml or system resource files. For example, you may want to override a JDBC Datasource xml module Username, Password, and URL so that it references a local database. 
+Use configuration overrides (also called _situational configuration_) to customize a WebLogic domain home configuration without modifying the domain's actual `config.xml` or system resource files. For example, you may want to override a JDBC datasource XML module user name, password, and URL so that it references a local database.
```

Comment:
I am told that "situational config" is not used externally. @rjeberhard can you confirm?

Reply:
Just so happens that I wrote the public doc for the feature. See https://docs-uat.us.oracle.com/en/middleware/weblogic-server/19.1/domcf/changes.html#GUID-3C5A2D0B-437C-41F1-B914-81F0184985CC  Look at the second para, third sentence.

Reply:
ok. cool by me.  i don't know a thing about it :) i was just told that

Reply:
The feature is called "Configuration Overriding" but it is referenced as situational configuration within the doc because many of the files names use the term.

Reply:
Also pretty much all of the config override WL Server log messages use the term 'situational' - at least the ones I've seen far.    It's a nice thing to know - makes it easy to pick them out fo the clutter.